templates/nomad.systemd.erb

```diff
@@ -1,18 +1,16 @@
 [Unit]
-Description=Nomad Agent
-Wants=basic.target
-After=basic.target network.target
+Description=Nomad agent
+Requires=network-online.target
+After=network-online.target
 
 [Service]
-User=<%= scope.lookupvar('nomad::user') %>
```

Comment:
I'm not super familiar with systemd init scripts. Can you describe why removing this was necessary? Same with the line below.